### PR TITLE
PythonLab: update delete button color on predict levels 

### PR DESCRIPTION
**Pull request by Erin007**
[CT-681](https://codedotorg.atlassian.net/browse/CT-681)

**Before `PredictResetButton`** 
Screenshot 2024-08-01 at 12 17 28 PM

**After `PredictResetButton`** 
Screenshot 2024-08-01 at 12 18 14 PM

**Before `ContainedLevelResetButton`**
Screenshot 2024-08-01 at 2 46 30 PM

**After `ContainedLevelResetButton`**

Screenshot 2024-08-02 at 12 44 28 PM

### CODE REVIEW

File: apps/src/templates/instructions/ContainedLevelResetButton.jsx [hunk 1]
```diff
@@ -42,6 +42,7 @@ export const UnconnectedContainedLevelResetButton = ({
   return (
     <div>
       <Button
+        name="containedLevelResetButton"
```
Comment (molly-moen) [inline apps/src/templates/instructions/ContainedLevelResetButton.jsx:45]: did you intend to change the button color on old contained levels too?

Comment (Erin007) [reply apps/src/templates/instructions/ContainedLevelResetButton.jsx:45]: Oh! I did, but maybe I'm not supposed to? That button was already red, but I updated it to use the newer DSCO `Button` component.

Comment (molly-moen) [reply apps/src/templates/instructions/ContainedLevelResetButton.jsx:45]: I think it's fine to update it! Can you add a before/after screenshot to the description?

**Comment by Erin007** [comment]
@molly-moen Do you want the `PredictResetButton` to be red (like prod) or to have a red outline (like the screenshot in the PR description)?

**Comment by molly-moen** [comment]
> @molly-moen Do you want the `PredictResetButton` to be red (like prod) or to have a red outline (like the screenshot in the PR description)?

I think we want the red outline, that's what Molly P recommended: https://codedotorg.slack.com/archives/C0532LCK125/p1720027891930459?thread_ts=1720026189.728519&cid=C0532LCK125
It looks like there's already a task for teacher tools to update the other button, so you could either skip those changes or connect with them!

**Comment by Erin007** [comment]
@molly-moen I got [confirmation ](https://codedotorg.slack.com/archives/C045UAX4WKH/p1722616428531169) from @MollyPeredo that secondary styling for the `ContainedLevelResetButton` is the change we want. https://codedotorg.slack.com/archives/C045UAX4WKH/p1722616428531169

**Review by molly-moen — APPROVED** [review]
Nice!

**Pull request merged**